.github/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+X. Enhancement - Add EGPG SMR - thanks to @JackG2004 (Jack Gil)
```

Comment:
```suggestion
X. Enhancement - Add Cumbernauld (EGPG) SMR - thanks to @JackG2004 (Jack Gil)
```
With the changelog we provide the name as well as the ICAO so it's a bit easier to understand